guides/v3.4.0/object-model/index.md

```diff
@@ -1,18 +1,21 @@
-You'll notice standard JavaScript class patterns and the new ES2015
-classes aren't widely used in Ember. Plain objects can still be found,
-and sometimes they're referred to as "hashes".
+One of the first things you'll notice when you generate JavaScript files in Ember is that most of the code you will write goes inside of an object. While an Ember Object might look a lot like an ES2015 JavaScript class, it has some special properties.
 
-JavaScript objects don't support the observation of property value changes.
-Consequently, if an object is going to participate in Ember's binding
-system you may see an `Ember.Object` instead of a plain object.
+### Introducing: Ember Objects
 
-[Ember.Object](https://www.emberjs.com/api/ember/release/modules/@ember%2Fobject) also provides a class system, supporting features like mixins
-and constructor methods. Some features in Ember's object model are not present in
-JavaScript classes or common patterns, but all are aligned as much as possible
-with the language and proposed additions.
+The [Ember Object](https://www.emberjs.com/api/ember/release/classes/EmberObject) class extends plain JavaScript objects to provide core framework functions such as participating in Ember's binding system.
```

Comment:
Can we add a very short definition of "binding system" here? Something like "... binding system, where changes to an object automatically trigger updates to the user interface."

Reply:
Added this as well